RFIDForIoT/CS/PiezoBuzzerLib/Buzzer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Devices.Gpio;
+
+namespace PiezoBuzzerLib
+{
+    public class Buzzer
+    {
+        public const int PIEZO_PIN = 24; // CONNECT TO 32th PIN OF upboard

```

Comment:
"32nd"  not  "32th"
"UP Board" not "upboard"

Reply:
Updated the changes